manifest.webapp

```diff
@@ -114,5 +114,12 @@
       "selector": "worker",
       "values": ["zip"]
     }
+  },
+  "acceptFromFlagship": true,
+  "accept_documents_from_flagship": {
+    "accepted_mimes": "[*] / [jpg,png,pdf]",
+    "number_of_files_max": 2,
+    "maxSizePerFile": "10Mo",
```

Comment:
@Ldoppea Do you already know the format you'll use?

Reply:
Nope, we should define it together.

`accepted_mimes` should contain the accepted file formats that the cozy-app can handle. If it handles every file formats then put `[*]`, else put the extensions in an array `[jpg,png,pdf]`. However maybe we should fit to the JSON notation here, so don't use quotes around the array.
Proposal:
```json
{
  "accepted_mimes": ["*"] // handle every file formats
}
```
or
```json
{
  "accepted_mimes": ["jpg", "png"] // handle only jpg and png
}
```

for `maxSizePerFile` i'm wondering if we should enforce the unit to "Mo" or not so we can ease the parsing. I'm not sure if we will ever limit a file size using Go notation.
Proposal:
```json
{
  "max_size_per_file_in_mo": 10 // limits to 10mo
}
```
or
```json
{
  "max_size_per_file_in_mo": 0 // no limit
}
```

What do you think about this?

Reply:
Also be sure we use the same convention than for other files (either all camlCase, or all snakeCase)

Reply:
@Crash-- @Ldoppea quel est votre ultime bafouille afin que je puisse modifier ?

Reply:
I confirm that the functional mime types are: `["application/pdf", "image/jpg", "image/jpeg", "image/png"]`